docs/tutorial.md

```diff
@@ -176,17 +176,18 @@ The final result of all the changes is this:
 {% include argo_example.cpp %}
 ```
 
-We can now compile the application. We assume that `${ARGO_INSTALL_DIRECTORY}` is
-where you installed ArgoDSM. If the ArgoDSM files are already in your
-`(LD_)LIBRARY_PATH` and include path, you can skip the `-L...` and `-I...`
-switches. If you have no idea what we are talking about, you should ask your
-system administrator.
+We can now compile the application. We assume that `${ARGO_INSTALL_DIRECTORY}`
+is where you installed ArgoDSM. If the ArgoDSM files are already in your
+`LIBRARY_PATH`, `INCLUDE_PATH`, and `LD_LIBRARY_PATH`, you can skip the
+`-L...`, `-I...`, and `-Wl,-rpath,...` switches. If you have no idea what we
```

Comment:
The order of the environment variables and compiler flags should ideally match the comment in docs/index.md. I'm not particular about which place changes.

Reply:
Order altered to match in docs/index.md by @davidklaftenegger.